### ISSUE TITLE
Project tools' path is not found and may include strange string

### ISSUE THREAD

**Issue by PrsPrsBK** [volta-cli/volta#444]
Hello. After `volta install typescript@3.1`, the path of `tsc` is not found.

```
pwsh:$ tsc --version
指定されたパスが見つかりません。# This means 'Path not Found', on jp locale.
```

I tried some `volta which`, and only path of `tsc` starts with `\\?\C:\`.
If that is not relevant, excuse me. Thanks in advance.

```
pwsh:$ volta which node
C:\Users\ppbk\AppData\Local\Volta\tools\image\node\8.16.0\6.4.1\node.EXE

pwsh:$ volta which npm
C:\Users\ppbk\AppData\Local\Volta\tools\image\node\8.16.0\6.4.1\npm.CMD

pwsh:$ volta which tsc
\\?\C:\Users\ppbk\AppData\Local\Volta\tools\image\packages\typescript\3.1.6\bin\tsc
```

Volta: ver.0.5.1 (installer)
OS: Microsoft Windows [Version 10.0.17763.475]

**Comment by chriskrycho** [comment]
Thank you so much for the detailed and helpful report! One of us will try to dig into this sometime soon!

**Comment by charlespierce** [comment]
Hi @PrsPrsBK, thanks for this report! It looks like in a refactor we lost some code that handles launching tools in Windows. I've opened a PR #445 to restore that code and get Windows working again. I'll make sure that's included in the next release (on Monday, May 20)

**Comment by charlespierce** [comment]
@PrsPrsBK We just published Volta `0.5.2`, which has a fix for this issue, so if you upgrade to the latest version it should work now.

**Comment by PrsPrsBK** [comment]
Thanks! Now `tsc --version` woks.
